src/features/profile/components/PersonDetailsCard.tsx

```diff
@@ -86,6 +86,11 @@ const PersonDetailsCard: React.FunctionComponent<{
         value = (
           <PersonDetailLink href={value as string}>{value}</PersonDetailLink>
         );
+      } else if (value && field.type == 'enum_text' && field.enum_choices) {
```

Comment:
This should be using the `CUSTOM_FIELD_TYPE` enum, not hard-coded text.

Reply:
fixed